src/http/api/webhook.controller.ts

```diff
@@ -128,6 +136,22 @@ export class WebhookController {
     }
 
     async handlePostDeleted(ctx: AppContext) {
+        let uuid: string;
+        try {
+            uuid = PostDeletedWebhookSchema.parse(
+                (await ctx.req.json()) as unknown,
+            ).post.previous.uuid;
+        } catch (err) {
+            if (err instanceof Error) {
+                return BadRequest(`Could not parse payload: ${err.message}`);
+            }
+            return BadRequest('Could not parse payload');
+        }
+
+        const account = ctx.get('account');
+
+        await this.ghostPostService.deleteGhostPost(account, uuid);
```

Comment:
Do we need to check if this op was ok and return appropriate res or are we ok always returning 200?

Reply:
I initially thought that since these are webhook URLs and no one is consuming the responses, returning 200 would be just fine. But now when I think of it, it makes sense to add appropriate responses after all. This way, we can have the responses in our logs, which could be useful for monitoring, metrics etc. Added here: https://github.com/TryGhost/ActivityPub/pull/1040/commits/0ceb53712156ace2d8376c21b4d51745d2cb7296

I'll also add these error responses for the other method(`updateArticleFromGhostPost`) separately for consistency.